pkg/sql/schemachanger/scplan/internal/scgraph/edge.go

```diff
@@ -102,6 +95,10 @@ const (
 	// be reached before the destination (to), and _must_ do so in a previous
 	// stage.
 	PreviousStagePrecedence
+
+	// PreviousTransactionPrecedence is like PreviousStagePrecedence but does
+	// not allow the transition to occur if the current phase is not PostCommit.
```

Comment:
Can you be elobarate a bit more on the comment? Does it mean`from` must be reached before `to` in a previous stage, and both stages must be in PostCommit phase? If so, what do you think of the name `PreviousStageInPostCommitPrecendence`?

Reply:
`does not allow the transition to occur unless the current phase is at least PostCommitPhase, because StatementPhase and PreCommitPhase are special in that they take place in the same transaction`?